packages/react-vapor/src/components/refresh/examples/RefreshExamples.tsx

```diff
@@ -28,6 +29,36 @@ export class RefreshExamples extends React.Component<any, any> {
                         </div>
                     </LabeledInput>
                 </Section>
+                <Section level={2} title="Component with a button to call the refreshCallback">
+                    <LabeledInput label="Refresh counter">
+                        <Button
+                            name="Start"
+                            className="btn mr2"
+                            onClick={() => ReactVaporStore.dispatch(RefreshCallBackActions.start('refresh-2'))}
```

Comment:
The example could be connected instead of using ReactVaporStore.dispatch yolo style.

Reply:
true, I can connect my example. I will